jest.config.js

```diff
@@ -129,7 +129,6 @@ module.exports = {
             "reportTestSuiteErrors": true
         }],
         ["jest-stare", {
-            "resultDir": "__tests__/__results__/jest-stare",
```

Comment:
Curious about the reason for this change? When the Jest Stare results are stored in a top-level folder, the coverage link breaks. I think we'd want to keep the old directory as the default value, and allow the `.env` files to override it.

Reply:
I'm going to move this into draft mode and wait for my other PR to be merged before merging that in here and addressing this.

Reply:
Addressed